python/nnvm/frontend/keras.py

```diff
@@ -289,6 +289,30 @@ def _convert_upsample(insym, keras_layer, _):
     return _sym.upsampling(insym, **params)
 
 
+def _convert_cropping(insym, keras_layer, _):
+    _check_data_format(keras_layer)
+    crop_type = type(keras_layer).__name__
+    if crop_type == "Cropping1D":
+        raise NotImplementedError("Cropping1D not implemented")
+    elif crop_type == "Cropping2D":
+        if isinstance(keras_layer.cropping, int):
+            top = bottom = keras_layer.cropping
```

Comment:
Better to use crop_t, crop_b, crop_l, crop_r to be consistent with conv and pool